site/gdocs/components/ArticleBlock.tsx

```diff
@@ -225,9 +225,9 @@ export default function ArticleBlock({
                 />
             )
         })
-        .with({ type: "simple-text" }, (block) => {
-            return block.value
-        })
+        .with({ type: "simple-text" }, (block) => (
+            <span>{block.value.text}</span>
```

Comment:
I think the point of the simple-text block is to [opt out of formatting altogether](https://github.com/owid/owid-grapher/blob/ce36882c8ac95d76fbad5d518cad8075ec855bea/db/model/Gdoc/rawToEnriched.ts#L1214-L1218), and leave that responsibility to the caller. "Regular" simple texts are already [wrapped in span in span-simple-text](https://github.com/owid/owid-grapher/blob/ce36882c8ac95d76fbad5d518cad8075ec855bea/site/gdocs/components/SpanElement.tsx#L15-L17).

Reply:
Good point, changed.